### PR TITLE
Add google Recatcha v3

### PR DESCRIPTION
**Pull request by duranmla**
**What:**
Allow to use google reCAPTCHA on donation endpoints

**Why:**
relates to https://github.com/debtcollective/home/pull/69

**How:**
follow ccdc881 approach for `/donate` and `/subscriptions`

**Extras:**
Fix issue with chapter "San Diego" to be a valid chapter

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -14,6 +14,7 @@ AMPLITUDE_API_KEY=API-KEY
 REDIS_URL=redis://127.0.0.1:6379
 SKYLIGHT_AUTHENTICATION=
 RUBYOPT='-W:no-deprecated -W:no-experimental'
+RECAPTCHA_SECRET_KEY_V3=
```
Comment (orlando) [inline .env.sample]: @duranmla is this one different from the other we have?

Also, do we need RECAPTCHA_SITE_KEY anymore? or we can remove it?

**Review by orlando — APPROVED** [review]
LGTM, small comment about environment variables If we need to remove the other keys I would like to do it before merging this

File: .env.sample (hunk 1 above)
Comment (duranmla) [reply .env.sample]: According to this https://developers.google.com/recaptcha/docs/faq I think that:

- If there is no other place using reCAPTCHA then we can remove what we used to have.
- We need a separate value key for the v3 reCAPTCHA

**Review by orlando — APPROVED** [review]
LGTM!

**Pull request merged**